src/Temporalio/Worker/Tuning/PollerBehavior.cs

```diff
@@ -0,0 +1,73 @@
+namespace Temporalio.Worker.Tuning
+{
+    /// <summary>
+    /// Defines the behavior of a poller.
+    /// </summary>
+    public abstract class PollerBehavior
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PollerBehavior"/> class.
+        /// </summary>
+        internal PollerBehavior()
+        {
+        }
+
+        /// <summary>
+        /// A poller behavior that will attempt to poll as long as a slot is available, up to the
+        /// provided maximum. Cannot be less than two for workflow tasks, or one for other tasks.
+        /// </summary>
+        public sealed class SimpleMaximum : PollerBehavior
+        {
+            /// <summary>
+            /// Initializes a new instance of the <see cref="PollerBehavior.SimpleMaximum"/> class.
+            /// </summary>
+            /// <param name="maximum">The maximum number of pollers at a time.</param>
+            public SimpleMaximum(uint maximum = 5)
```

Comment:
I don't think the constructor parameters here or in autoscaling one should have default values, same as they don't in https://github.com/temporalio/sdk-python/pull/830